.gitignore

```diff
@@ -13,12 +13,15 @@
 !/log/.keep
 !/tmp/.keep
 
-# Ignore uploaded files in development
+# Ignore pidfiles, but keep the directory.
+/tmp/pids/*
+!/tmp/pids/
+!/tmp/pids/.keep
```

Comment:
It would seem these 3 lines can be combined into one and moved up to the /log/ and /tmp/ section?